images/Dockerfile.ci

```diff
@@ -0,0 +1,17 @@
+FROM registry.svc.ci.openshift.org/origin/4.5:base
```

Comment:
just as you're aware that this means it's going to run rhel7, are we ok with this ?

Reply:
I'm still thinking about how best to integrate this, but the base container OS probably doesn't matter here.

CI jobs need a container, but for e2e-metal-ipi, it runs on Packet anyway outside the OCP cluster.  The options for testing dev-scripts in CI are basically (1) have a container with dev-scripts content, and scp it over to the Packet host, (2) just have the CI job look at the PULL_* environment variables from CI and git clone on packet directly.

Reply:
I think the CI folks do prefer option 1, since it's more like how everything else runs but I'm not sure it's worth it.